### PR TITLE
Move acme state tests to pytest

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Move acme state tests to pytest

### CODE REVIEW

File: tests/pytests/unit/states/test_acme.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            "name": "test",
+            "result": True,
+            "comment": ["Certificate test exists and does not need renewal."],
+            "changes": {},
+        }
+        assert acme.cert("test") == match
+        assert acme.cert("testing.example.com", certname="test") == match
+
+
+def test_cert_fresh_certificate_t():
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_acme.py:57]:
```suggestion
def test_cert_fresh_certificate_test():
```

File: tests/pytests/unit/states/test_acme.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            "name": "test",
+            "result": True,
+            "comment": ["Mockery"],
+            "changes": {"new": {"foo": "bar"}},
+        }
+        assert acme.cert("test") == match
+        assert acme.cert("testing.example.com", certname="test") == match
+
+
+def test_cert_renew_certificate_t():
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_acme.py:101]:
```suggestion
def test_cert_renew_certificate_test():
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/states/test_acme.py (hunk 1 above)
Comment (krionbsd) [reply tests/pytests/unit/states/test_acme.py]: Fixed

File: tests/pytests/unit/states/test_acme.py (hunk 2 above)
Comment (krionbsd) [reply tests/pytests/unit/states/test_acme.py]: Fixed

**Review by s0undt3ch — APPROVED** [review]

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**